pages/DSNP/UserData.md

```diff
@@ -0,0 +1,211 @@
+# User Data
+
+DSNP stores user-related data of well defined formats associated with a user's DSNP Identifier.
+DSNP User Data is designed for data with the following characteristics:
+* The data is intended to be accessible across the network with the strongest possible durability guarantees
+* The data may change frequently.
+* The data does not need to be widely announced and instead can be read just-in-time for a particular user.
+
+Data may be public or stored in encrypted form.
+The [Avro](https://avro.apache.org) schema specification is used to define the binary representation of relevant data types.
+
+## User Data Types
+
+DSNP implementations MUST support the following user data types:
+
+| System Name | Version | Encryption Algorithm | Compression Codec | Avro Object Type |
+| --- | --- | --- | --- | --- |
+| `publicFollows` | 1.2 | NONE | [`brotli`](https://brotli.org) | [GraphEdge](Graph.md#edges) |
```

Comment:
I  thought we decided to go with Deflate?

Reply:
I just haven't rebased yet.